source/module_hamilt_lcao/hamilt_lcaodft/operator_lcao/td_ekinetic_lcao.h

```diff
@@ -92,17 +100,7 @@ class TDEkinetic<OperatorLCAO<TK,TR>> : public OperatorLCAO<TK, TR>
     /// @brief correction term A^2*S
     void td_ekinetic_grad(std::complex<double>* Hloc, int nnr, ModuleBase::Vector3<double> grad_overlap);
 
-    ORB_table_phi MOT;
-	  ORB_gaunt_table MGT;
-    
-    /// @brief Store the two center integrals outcome <𝝓_𝝁𝟎 |𝛁| 𝝓_𝝂𝑹> for td_ekinetic term
```

Comment:
How can you write the "<𝝓_𝝁𝟎 |𝛁| 𝝓_𝝂𝑹>" formulas? Can they be recoganized in other developers' environments?

Reply:
This is not latex type formula but some combination of special characters. I text them out in word or powerpoint and then copy them here. So I guess it can be recoganized normally with any editor which supports UTF-8 format text. 
Here is what it looks like in two basic editor: vi for linux and txt for windows.

Vi:
<img width="524" alt="vi" src="https://github.com/deepmodeling/abacus-develop/assets/104255052/d78768b5-5f3c-47c4-9f67-575739f8d936">
TXT:
<img width="976" alt="2" src="https://github.com/deepmodeling/abacus-develop/assets/104255052/5bfbe8f6-b0f4-46eb-8f47-b07e5ef87093">

I guess these special characters look much more clear than English characters.